docs/Testing.rst

```diff
@@ -91,6 +95,11 @@ Besides ``check-swift``, other targets are also available. Here's the full list:
 
   Runs all tests (primary, validation, and long).
 
+* ``SwiftUnitTests``
+
+  Builds all unit tests.  Executables are located under
+  ``${SWIFT_BUILD_ROOT}/unittests`` and must be run individually.
```

Comment:
I think it's worth mentioning that they *can* be run individually but are also run by `check-swift` by default (which they are).